server/server.go

```diff
@@ -1318,12 +1318,12 @@ func (s *Server) SetReplicationModeConfig(cfg config.ReplicationModeConfig) erro
 	return nil
 }
 
-// IsServing returns whether the server is the leader, if there is embedded etcd, or the primary otherwise.
+// IsServing returns whether the server is the leader if there is embedded etcd, or the primary otherwise.
 func (s *Server) IsServing() bool {
 	return s.member.IsLeader()
 }
 
-// AddServiceReadyCallback adds the callback function when the server becomes the leader, if there is embedded etcd, or the primary otherwise.
+// AddServiceReadyCallback adds a callback when the server becomes the leader if there is embedded etcd, or the primary otherwise.
```

Comment:
```suggestion
// AddServiceReadyCallback adds callback functions when the server becomes the leader if there is embedded etcd, or the primary otherwise.
```

Reply:
changed. other places too